pom.xml

```diff
@@ -540,6 +540,12 @@ Copyright 2014 CyberVision, Inc.
                 <version>${project.version}</version>
             </dependency>
             <dependency>
+                <groupId>junit</groupId>
```

Comment:
Why are you editing main pom.xml? Why moving same dependency from one place to another?